clouddriver-oracle/src/main/groovy/com/netflix/spinnaker/clouddriver/oracle/service/servergroup/OracleServerGroupPersistence.groovy

```diff
@@ -211,7 +212,14 @@ class OracleServerGroupPersistence {
           String json
           inputStream.withStream { json = inputStream.getText("UTF-8") }
           sg = jsonToServerGroup(json, ctx.creds)
-          return sg
+          return sg 
+        } catch (BmcException e) {
+          if (e.getStatusCode() == 404) {
+            log.warn(e.getLocalizedMessage())
+          } else {
+            log.error("OSS Read exception", e)
```

Comment:
Any value in propagating this to the user?

Reply:
I saw the 'e' parameter may produce stacktrace in the log, is that what you mean?
I remove the e but log the e.getLocalizedMessage().
Or did you mean I should change from error to warning or nothing?